### ISSUE TITLE
[Request] ESXI-Arm Fling Support on 64 Bit Nightlies

### ISSUE THREAD

**Issue by MarkProminic** [guysoft/OctoPi#725]
#### Goal:

I have currently setup the Raspberry Pi 4 that I was going to use for my  Ender 3 V2 using the ESXI-Arm fling for other testing and development purposes. I was able to get the OctoPi to boot (mostly) following these instructions

#### References:
https://www.virtuallyghetto.com/2020/10/how-to-run-raspberry-pi-os-as-a-vm-on-esxi-arm.html
https://www.virtuallyghetto.com/2020/11/how-to-run-kali-linux-on-esxi-arm.html

#### Issue Description:
I eventually got to the OS booting, but it stops at an Initramfs prompt.  I tried to install linux-image-arm64, but it doesn't appear to be available for Ubuntu 20.10 for Arm, I am likely using the wrong package, and I have a hunch that if I get the right one, following the steps in the above guide, plus using the right kernel image, would get me to being able to boot this in ESXI-Arm:

#### Additional Means of getting this to work:

I tried to use the Vagrant, but there isn't a Vagrant Box for ESXI, only Virtualbox, So I tried to create a Packer build with the VMWare-ISO builder to use a base image for the Vagrant process, however it appears that since it currently does not allow one to specify the Architecture as Arm64, you are unable to initiate the build. 

I have submitted a feature request with Packer to include support for this, for rapid deployment and testing of any Arm64 based OSs and software.

https://github.com/hashicorp/packer/issues/10706

#### Version of OctoPi:

64 Bit Experimental Nightly Build -- I believe that Arm Fling can only run  64bit arm currently.

#### Printer model & used firmware incl. version

N/A

**Comment by MarkProminic** [comment]
Hello,

I have actually got this to run, I used the wrong Nightly release. I wasn't sure what the difference was between:

2021-01-11_2020-08-20-octopi-buster-arm64-lite-0.18.0.zip

and 

2021-02-28_octopi-20.10-preinstalled-server-arm64+raspi-1.0.0.zip

with: 2021-01-11_2020-08-20-octopi-buster-arm64-lite-0.18.0.zip

 I was able to get OctiPi up and working on ESXI-Arm for Raspberry Pi. I will followup with more details.
